Libs/MRML/Logic/vtkMRMLColorLogic.h

```diff
@@ -93,7 +144,7 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLColorLogic : public vtkMRMLAbstractLogic
   /// Return a default color node id for a chart
   virtual const char * GetDefaultChartColorNodeID();
 
-  /// Add a file to the input list list, checking first for null, duplicates
+  /// Add a file to the input list Files, checking first for null, duplicates
   void AddColorFile(const char *fileName, std::vector<std::string> *Files);
```

Comment:
Why a pointer ? That means you will have to manage the memory ... would that work:

```
void AddColorFile(const std::string& fileName, const std::vector<std::string>& Files);
```